### PR TITLE
Without design

### PR DESCRIPTION
**Pull request by vadimdom**

### CODE REVIEW

File: client/src/components/pages/profile-page/ProfilePage.jsx [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+  }
+
+  handleSubmitChanges = () => {
+    this.props.updateUser(
+      this.state.id,
+      this.state.newname,
+      this.state.newsecondName,
+      this.state.newlogin,
+    );
+    this.logout();
```
Comment (Shaonari) [inline client/src/components/pages/profile-page/ProfilePage.jsx]: Should we logout user after updating information?

File: client/src/components/pages/profile-page/ProfilePage.jsx [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    this.props.logout(refToken);
+  }
+
+  handlePasswordChange = () => {
+    this.setState({
+      newpassword: '',
+    });
+  }
+
+  render() {
```
Comment (Shaonari) [inline client/src/components/pages/profile-page/ProfilePage.jsx]: create methods for rendering each type of content

File: client/src/redux/modules/auth/actions.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      });
+  };
+}
+
+export const verifyCredentials = () => (dispatch) => {
+  const accessToken = localStorage.getItem('AccessToken');
+  const refreshToken = localStorage.getItem('RefreshToken');
+  const currentUser = JSON.parse(localStorage.getItem('User'));
+
+  if (accessToken && refreshToken && currentUser) {
```
Comment (Shaonari) [inline client/src/redux/modules/auth/actions.js]: user unlikely needs here

File: client/src/redux/modules/posts/actions.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+      .catch((err) => {
+        console.error(err);
+        throw err;
+      });
+  }
+}
+
+export function updatePost(Title, Description, idPost) {
+  return (dispatch) => {
+    return axiosRequest.patch(`/api/posts`, {idPost, Description, Title})
```
Comment (Shaonari) [inline client/src/redux/modules/posts/actions.js]: security lack(will describe manually)

File: client/src/redux/modules/posts/actions.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    .catch((err) => {
+      console.log(err);
+      throw err;
+    });
+  }
+}
+
+export function deletePost(idPost) {
+  return (dispatch) => {
+    return axiosRequest.delete(`/api/posts/${idPost}`, idPost)
```
Comment (Shaonari) [inline client/src/redux/modules/posts/actions.js]: security lack

File: client/src/redux/modules/profile/actions.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+        console.error(err);
+        throw err;
+      });
+  };
+}
+
+export function updateUser(userId, FirstName, LastName, Login ) {
+  return (dispatch) => {
+    return axiosRequest
+      .patch(`/api/users/${userId}`, { FirstName, LastName, Login } )
```
Comment (Shaonari) [inline client/src/redux/modules/profile/actions.js]: security lack

File: client/src/components/common/footer/index.jsx [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
```
[the remark this answers is not in the document]
Comment (DinAlla) [reply client/src/components/common/footer/index.jsx]: It doesn't work without React

File: client/src/components/pages/addpost-page/AddPostPage.jsx [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+    } 
+  }
+
+  handleInputChange = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  }
+
+  handleAddPost(title, description) {
```
[the remark this answers is not in the document]
Comment (DinAlla) [reply client/src/components/pages/addpost-page/AddPostPage.jsx]: It's better to hang up an action check. Not in component

**Pull request merged**